### PR TITLE
Complete documentation of type attributes

### LINKED ISSUE #1532: Document efipartsize parameter in <type> settings.
## Problem description
Documentation for  efipartsize parameter in <type> settings is missing in:
https://osinside.github.io/kiwi/image_description/elements.html#preferences-type

## Expected behaviour
Please add it in documentation.

## Steps to reproduce the behaviour
documentation issue.

## OS and Software information
+ KIWI version: 
+ Operating system: 
+ OBS version:

### PR DESCRIPTION
**Pull request by schaefi**
Some type attributes e.g efipartsize and others were not
documented properly. This Fixes #1532

### CODE REVIEW

File: doc/source/image_description/elements.rst [hunk 1]
```diff
@@ -406,6 +421,190 @@ target_blocksize="number"
...
+  Attributes can be specified as comma separated list.
+  Currently the attributes `no-copy-on-write` and `synchronous-updates`
+  are available. Can only be configured for the disk image
+  types oem and vmx
+
+spare_part_is_last="true|false":
+  Specify if the spare partition should be the last one in
+  the partition table. Can only be configured for the vmx
+  disk image type. By default the root partition is the last
+  one and the spare partition lives behind it. With this
```
Comment (davidcassany) [inline doc/source/image_description/elements.rst:458]: nit: probably this is a misunderstanding caused by my poor English, to me it is odd having a partition behind last one... shouldn't it be before? To my understanding if something is last, there isn't anything else behind it, just before it :P

**Review by davidcassany — APPROVED** [review]
looks good to me, just got slightly confused by one sentence, but probably this is a result of my poor English

File: doc/source/image_description/elements.rst (hunk 1 above)
Comment (schaefi) [reply doc/source/image_description/elements.rst:458]: Surely not your English but me writing a wrong sentence :) Thanks for pointing this out. Of course if by default root is last, spare lives before not behind. I will fix this

Comment (schaefi) [reply doc/source/image_description/elements.rst]: hope it makes more sense now :)

**Pull request merged**